apps/authentication/views.py

```diff
@@ -89,6 +90,9 @@ def register(request):
                 email.primary = True
                 email.save()
 
+                # Set default privacy settings
+                Privacy.objects.create(user=user)
+
```

Comment:
Users can also sign up via the api (POST to `/api/v1/users/`), which this would not cover.

What about doing this as a `post_save` signal on OnlineUser when the `created` flag is `True`? That would cover both cases 😄

Reply:
Like this? 😏 https://github.com/dotkom/onlineweb4/blob/develop/apps/profiles/signals.py#L10

Reply:
I looked a little and it and I think that signal might be a bit buggy because it doesn't work for new users until they click My Page. Might be that the Signal had to be connected to OnlineUser

Reply:
"min side" view has a hack which does this, so that might be the case. I'm positive it _has_ worked before though, as we had problems with this before.